### ISSUE TITLE
nsg_list is case sensitive and the list variable is created as NSG_list on network_config.sh

### ISSUE THREAD

**Issue by 105472229-myseneca** [Azure-Project-Winter2024/Azure-Project-Scripts#2]
https://github.com/Azure-Project-Winter2024/Azure-Project-Scripts/blob/c5865f50ffe3b9ca7850f3eca17cf0487708bd66/bash-scripts/delete_all_nsg.sh#L13

@AtoosaNasiri-Seneca 

Recommended change as variable names are case sensitive:
```
for nsg_name in "${NSG_list[@]}"
```

**Comment by AtoosaNasiri-Seneca** [comment]
@105472229-myseneca 

That is correct, bash is case-sensitive for all parameters and variables, that i why I have recommended you **not** to make any modifications in the scripts for now, until you are more expert and know how to make changes without breaking the code

**Comment by 105472229-myseneca** [comment]
I recommended this change as on the backend_config.sh the variable has been declared with uppercase NSG instead of lowercase. 

https://github.com/Azure-Project-Winter2024/Azure-Project-Scripts/blob/c5865f50ffe3b9ca7850f3eca17cf0487708bd66/bash-scripts/backend_config.sh#L70

And when delete_all_nsg.sh calls the list, it calls it lowercase. 

https://github.com/Azure-Project-Winter2024/Azure-Project-Scripts/blob/c5865f50ffe3b9ca7850f3eca17cf0487708bd66/bash-scripts/delete_all_nsg.sh#L13
